sorted_words.rb

```diff
@@ -0,0 +1,18 @@
+# Write a program called sorted_words.rb. 
+# It should prompt the user for words and add each to an array. 
+# The user should be able to add as many words as they like, until they just hit enter to return a blank word. 
+# Then sort the array using the sort method and print it out.
+
+words = []
+
+    print "Enter as many words (or press Enter to finish)? "
```

Comment:
I tested this method and unfortunately it is not working. I think it would be helpful to add a loop here and ask for one word at a time, push that one word into the words array, and keep looping until you need to `break` the loop.

On line 8, you are currently asking for multiple words at once, so when you push the words it looks like this:
`words.push("zoo apple dog cat")` which means `words = ["zoo apple dog cat"]` 

It would be helpful to sort if `words=["zoo", "apple", "dog", "cat"]`